internal/event/event.go

```diff
@@ -34,9 +34,10 @@ type Event struct {
 }
 
 const (
-	TypeState           = "state"
-	TypeAcknowledgement = "acknowledgement"
-	TypeInternal        = "internal"
+	TypeState                 = "state"
+	TypeAcknowledgement       = "acknowledgement"
+	TypeReevaluationScheduled = "reevaluation-scheduled"
```

Comment:
For this one, the relevant thing that happened, is that the incident reached a certain age, so I'd say that should be reflected in the type.

Reply:
I have renamed it to reflect it being scheduled due to the age.